.env.template

```diff
@@ -17,3 +17,6 @@ DISCORD_SERVER_ID=
 DISCORD_BOT_COMMAND_CHANNEL_ID=
 DISCORD_VOICE_CHANNEL_ID=
 DISCORD_WEBHOOK_URL=
+
+DISABLE_GPU=false
+DISABLE_SOFTWARE_RASTERIZER=false
```

Comment:
I would prefer to have only one environnent variable, for an everyday user the key `DISABLE_SOFTWARE_RASTERIZER` would make no sense to them.

Reply:
Please add the entry to `typings\module.d.ts`.

Reply:
I stick with POSIX standard, please add the last empty line back.